### PR TITLE
Added the border bottom of the modal's header

### LINKED ISSUE #33720: Modal header border bottom is missing
### Prerequisites

- [X] I have carried out troubleshooting steps and I believe I have found a bug.
- [X] I have searched for similar bugs in both open and closed issues and cannot find a duplicate.

### Describe the bug

This is related to https://github.com/woocommerce/google-listings-and-ads/issues/1583.

When we open a Modal component, it is missing the border / separator between the modal header title and modal body content. The border is missing because of some CSS from the WooCommerce core:

image

It seems like the issue is coming from the following lines: 

https://github.com/woocommerce/woocommerce/blob/4707dcbec08e93713fb7374e8f33f3a1b45dbec5/plugins/woocommerce-admin/client/tasks/tasks.scss#L55-L59

I tested previous WC versions and I found that the issue happens in WC 6.6.0 and 6.5.0 too.

In WC 6.4.0, the modal component shows up fine, with a border / separator between the modal header title and modal content:

image

### Expected behavior

There should be a border between the modal header and modal body, similar to @wordpress/components modal: https://wordpress.github.io/gutenberg/?path=/story/components-modal--default

image

### Actual behavior

There is no border between the modal header and modal body.

### Steps to reproduce

Refer to https://github.com/woocommerce/google-listings-and-ads/issues/1583. 

Alternatively, you could also create a dummy modal component in WooCommerce. The modal component would show up with no border / separator between modal header and modal body by default. See an example below, where I create a simple test modal in WooCommerce > Marketing page:

image

### WordPress Environment

<details>
	<summary>Click to expand</summary>

WordPress address (URL): https://wc67compattest.test
Site address (URL): https://wc67compattest.test
WC Version: 6.7.0
REST API Version: ✔ 6.7.0
WC Blocks Version: ✔ 7.8.3
Action Scheduler Version: ✔ 3.4.0
Log Directory Writable: ✔
WP Version: 6.0
WP Multisite: –
WP Memory Limit: 2 GB
WP Debug Mode: –
WP Cron: ✔
Language: en_US
External object cache: –

### Server Environment ###

Server Info: nginx/1.23.0
PHP Version: 8.0.20
PHP Post Max Size: 512 MB
PHP Time Limit: 30
PHP Max Input Vars: 1000
cURL Version: 7.84.0
(SecureTransport) OpenSSL/1.1.1p

SUHOSIN Installed: –
MySQL Version: 8.0.29
Max Upload Size: 512 MB
Default Timezone is UTC: ✔
fsockopen/cURL: ✔
SoapClient: ✔
DOMDocument: ✔
GZip: ✔
Multibyte String: ✔
Remote Post: ✔
Remote Get: ✔

### Database ###

WC Database Version: 6.7.0
WC Database Prefix: wp_
Total Database Size: 7.04MB
Database Data Size: 4.73MB
Database Index Size: 2.31MB
wp_woocommerce_sessions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_api_keys: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_attribute_taxonomies: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_downloadable_product_permissions: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_order_items: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_order_itemmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_tax_rates: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_woocommerce_tax_rate_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zones: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_shipping_zone_locations: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_shipping_zone_methods: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_woocommerce_payment_tokens: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_woocommerce_payment_tokenmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_woocommerce_log: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_actions: Data: 0.06MB + Index: 0.11MB + Engine InnoDB
wp_actionscheduler_claims: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_groups: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_actionscheduler_logs: Data: 0.05MB + Index: 0.03MB + Engine InnoDB
wp_automatewoo_abandoned_carts: Data: 0.02MB + Index: 0.08MB + Engine InnoDB
wp_automatewoo_customer_meta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_automatewoo_customers: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_automatewoo_events: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_automatewoo_guest_meta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_automatewoo_guests: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_automatewoo_log_meta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_automatewoo_logs: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_automatewoo_queue: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_automatewoo_queue_meta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_commentmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_comments: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_gla_budget_recommendations: Data: 0.22MB + Index: 0.14MB + Engine InnoDB
wp_gla_merchant_issues: Data: 0.06MB + Index: 0.00MB + Engine InnoDB
wp_gla_shipping_rates: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_gla_shipping_times: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_links: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_options: Data: 3.03MB + Index: 0.09MB + Engine InnoDB
wp_postmeta: Data: 0.16MB + Index: 0.13MB + Engine InnoDB
wp_posts: Data: 0.06MB + Index: 0.06MB + Engine InnoDB
wp_term_relationships: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_term_taxonomy: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_termmeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_terms: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_usermeta: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_users: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_admin_note_actions: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_admin_notes: Data: 0.05MB + Index: 0.00MB + Engine InnoDB
wp_wc_category_lookup: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_customer_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_download_log: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_coupon_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_order_product_lookup: Data: 0.02MB + Index: 0.06MB + Engine InnoDB
wp_wc_order_stats: Data: 0.02MB + Index: 0.05MB + Engine InnoDB
wp_wc_order_tax_lookup: Data: 0.02MB + Index: 0.03MB + Engine InnoDB
wp_wc_product_attributes_lookup: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_product_download_directories: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_product_meta_lookup: Data: 0.02MB + Index: 0.09MB + Engine InnoDB
wp_wc_rate_limits: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_reserved_stock: Data: 0.02MB + Index: 0.00MB + Engine InnoDB
wp_wc_tax_rate_classes: Data: 0.02MB + Index: 0.02MB + Engine InnoDB
wp_wc_webhooks: Data: 0.02MB + Index: 0.02MB + Engine InnoDB

### Post Type Counts ###

attachment: 23
aw_workflow: 2
page: 8
post: 2
product: 21
product_variation: 7
revision: 1
shop_coupon: 1
shop_order: 4

### Security ###

Secure connection (HTTPS): ✔
Hide errors from visitors: ✔

### Active Plugins (8) ###

AutomateWoo: by WooCommerce – 5.5.16
Google Listings and Ads: by WooCommerce – 1.13.6
WooCommerce Beta Tester: by WooCommerce – 2.0.5
WooCommerce Stripe Gateway: by WooCommerce – 6.4.3
WooCommerce Payments: by Automattic – 4.3.0
WooCommerce: by Automattic – 6.7.0-beta.2
WordPress Importer: by wordpressdotorg – 0.7
Mailhog for WordPress: by Tareq Hasan – 1.0.1

### Inactive Plugins (3) ###

Akismet Anti-Spam: by Automattic – 4.2.4
Hello Dolly: by Matt Mullenweg – 1.7.2
Relative URL: by Tunghsiao Liu – 0.1.8

### Settings ###

API Enabled: –
Force SSL: –
Currency: USD ($)
Currency Position: left
Thousand Separator: ,
Decimal Separator: .
Number of Decimals: 2
Taxonomies: Product Types: external (external)
grouped (grouped)
simple (simple)
subscription (subscription)
variable (variable)
variable subscription (variable-subscription)

Taxonomies: Product Visibility: exclude-from-catalog (exclude-from-catalog)
exclude-from-search (exclude-from-search)
featured (featured)
outofstock (outofstock)
rated-1 (rated-1)
rated-2 (rated-2)
rated-3 (rated-3)
rated-4 (rated-4)
rated-5 (rated-5)

Connected to WooCommerce.com: –
Enforce Approved Product Download Directories: ✔

### WC Pages ###

Shop base: #6 - /shop/
Cart: #7 - /cart/
Checkout: #8 - /checkout/
My account: #9 - /my-account/
Terms and conditions: ❌ Page not set

### Theme ###

Name: Storefront
Version: 4.1.0 (update to version 4.1.1 is available)
Author URL: https://woocommerce.com/
Child Theme: ❌ – If you are modifying WooCommerce on a parent theme that you did not build personally we recommend using a child theme. See: How to create a child theme
WooCommerce Support: ✔

### Templates ###

Overrides: –

### Subscriptions ###

WCS_DEBUG: ✔ No
Subscriptions Mode: ❌ Staging
Subscriptions Live URL: https://4e52-202-185-228-254.ap.ngrok.io
Subscription Statuses: –
WooCommerce Account Connected: ❌ No

### Store Setup ###

Country / State: United States (US) — California

### Payment Gateway Support ###

Cash on delivery: products

### Admin ###

Enabled Features: activity-panels
analytics
coupons
customer-effort-score-tracks
experimental-products-task
experimental-import-products-task
experimental-fashion-sample-products
experimental-product-tour
homescreen
marketing
mobile-app-banner
navigation
onboarding
onboarding-tasks
remote-inbox-notifications
remote-free-extensions
payment-gateway-suggestions
shipping-label-banner
subscriptions
store-alerts
transient-notices
wc-pay-promotion
wc-pay-welcome-page
wc-pay-subscriptions-page

Disabled Features: minified-js
settings

Daily Cron: ✔ Next scheduled: 2022-07-05 15:45:15 +00:00
Options: ✔
Notes: 50
Onboarding: skipped

### WooCommerce Payments ###

Version: 4.3.0
Connected to WPCOM: Yes
Blog ID: 208495287
Account ID: -

### Action Scheduler ###

Complete: 107
Oldest: 2022-07-01 11:21:34 +0000
Newest: 2022-07-04 15:56:06 +0000

Pending: 3
Oldest: 2022-07-05 03:00:00 +0000
Newest: 2022-07-05 14:21:45 +0000

### Status report information ###

Generated at: 2022-07-04 15:57:49 +00:00

</details>

### Isolating the problem

- [X] I have deactivated other plugins and confirmed this bug occurs when only WooCommerce plugin is active.
- [X] This bug happens with a default WordPress theme active, or [Storefront](https://woocommerce.com/storefront/).
- [X] I can reproduce this bug consistently using the steps above.

### PR DESCRIPTION
**Pull request by aezazs-multidots**
### All Submissions:

-   [x] Have you followed the [WooCommerce Contributing guideline](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md)?
-   [x] Does your code follow the [WordPress' coding standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/)?
-   [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change?

### Changes proposed in this Pull Request:

I have fixed the border-bottom not coming in the header of the component modal.

Closes #33720

### Steps to reproduce:

1. Use a fresh site
2. Go to OBW
3. Fill out store details
4. Click "Continue"
5. Observer that the `Build a better WooCommerce` modal's header contains a bottom border 

Screen Shot 2022-07-15 at 14 34 22

### Other information:

-   [x] Have you added an explanation of what your changes do and why you'd like us to include them?
-   [ ] Have you written new tests for your changes, as applicable?
-   [x] Have you successfully run tests with your changes locally?
-   [x] Have you created a changelog file for each project being changed, ie `pnpm changelog add --filter=<project>`?

### FOR PR REVIEWER ONLY:

-   [x] I have reviewed that everything is sanitized/escaped appropriately for any SQL or XSS injection possibilities. I made sure Linting is not ignored or disabled.

### CODE REVIEW

File: plugins/woocommerce/changelog/added-modal-border-bottom [hunk 1]
```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: enhancement
+
+Fixed 33720 - I have fixed the border-bottom not coming in the header of the component modal.
```
Comment (chihsuan) [inline plugins/woocommerce/changelog/added-modal-border-bottom]: suggestion ⬇️ 
```suggestion
Add the border bottom of the modal's header
```

**Comment by chihsuan** [comment]
@aezazs-multidots Could you also add test instructions for this PR? Thanks!

**Comment by aezazs-multidots** [comment]
Hello @chihsuan 

I have added the steps of `How to test the changes in this Pull Request:` in this PR. also I have done what you asked to change in the changelog entry.

Please let me know if you have any queries about the same.

Thanks & Regards

**Review by chihsuan — APPROVED** [review]
LGTM!

**Pull request merged**